fundamentals-2/script.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+///////////////////////////////////////
+// Coding Challenge #1
+
+/*
+Back to the two gymnastics teams, the Dolphins and the Koalas! There is a new gymnastics discipline, which works differently.
+Each team competes 3 times, and then the average of the 3 scores is calculated (so one average score per team).
+A team ONLY wins if it has at least DOUBLE the average score of the other team. Otherwise, no team wins!
+
+1. Create an arrow function 'calcAverage' to calculate the average of 3 scores
+2. Use the function to calculate the average for both teams
+3. Create a function 'checkWinner' that takes the average score of each team as parameters ('avgDolhins' and 'avgKoalas'), and then logs the winner to the console, together with the victory points, according to the rule above. Example: "Koalas win (30 vs. 13)".
+4. Use the 'checkWinner' function to determine the winner for both DATA 1 and DATA 2.
+5. Ignore draws this time.
+
+TEST DATA 1: Dolphins score 44, 23 and 71. Koalas score 65, 54 and 49
+TEST DATA 2: Dolphins score 85, 54 and 41. Koalas score 23, 34 and 27
+
+HINT: To calculate average of 3 values, add them all together and divide by 3
+HINT: To check if number A is at least double number B, check for A >= 2 * B. Apply this to the team's average scores 😉
+
+GOOD LUCK 😀
+*/
+
+const calcAverage = (a,b,c) => (a + b + c) / 3;
+
+const averageDolphins = calcAverage(85,54,71);
+const averageKoalas = calcAverage(23,34,27);
+
+function checkWinner (teamScore1, teamScore2){
+    if (teamScore1 >= (2 * teamScore2)){
+        console.log(`Dolphins win! (${teamScore1} - ${teamScore2})`)
+    }else if(teamScore2 >= (2 * teamScore1)){
+        console.log(`Koalas win! (${teamScore2} - ${teamScore1})`)
+    }else{
+        console.log("No one is the winner!")
+    }
+}
+
+checkWinner(averageDolphins,averageKoalas);
```

Comment:
EOF (means END OF FILE) we usually left the last line empty (you'll need to add extra whitespace)